spec/features/collection_facets/collection_facets_spec.rb

```diff
@@ -246,6 +246,26 @@
     end
   end
 
+  context "when applied science keyword facets and search terms filter the collections list to no results" do
+    before(:all) do
+      find(".facets-item", text: "BIOSPHERE").click
+      fill_in :keywords, with: "somestringthatmatchesnocollections"
+      wait_for_xhr
+    end
+
+    after(:all) do
+      reset_search
+      find(".facets-item", text: 'BIOSPHERE').click
```

Comment:
Doesn't reset_search reset the facets list, too? If so, performing this click could apply the biosphere facet and break subsequent tests.